cli/src/declarations.ts

```diff
@@ -453,6 +453,16 @@ export interface CapacitorConfig {
      * @default true
      */
     handleApplicationNotifications?: boolean;
+
+    /**
+     * On iOS 16.4 and greater, always enable debuggable web content.
+     *
+     * This is automatically enabled during development.
```

Comment:
Can we change this phrasing just a bit to say: 

```
* On iOS 16.4 and greater, enable debuggable web content for release builds.
*
* This is always `true` for development builds.
```

Just to help clarify that this flag ONLY affects release builds and that inspectable views are always true for non-release builds.